lib/fluent/gelf_util.rb

```diff
@@ -89,6 +89,16 @@ def make_gelfentry(tag,time,record, conf = {})
 
     end
 
+    def is_text(bytes)
```

Comment:
Isn't there a ruby built-in for that? Can you add a comment to this function what it's doing exactly. When data you expect to come to return false? According to unit tests, you care about whitespace, which can be simply removed removed with `.strip`, as suggested above.

Reply:
Right! I wanted to catch all the whitespace-like characters. It seems that [strip](https://apidock.com/ruby/String/strip) can do that pretty well.